<ui version="4.0" >
 <class>HMMSearchDialog</class>
 <widget class="QDialog" name="HMMSearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>222</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>hmm_search_dialog_title</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>hmm_file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hmmFileEdit" />
     </item>
     <item>
      <widget class="QToolButton" name="hmmFileButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="expertOptionsBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>expert_box_title</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="toolTip" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="statusTip" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="whatsThis" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="text" >
         <string>results_evalue_cutoff</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>319</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>319</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="toolTip" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="statusTip" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="whatsThis" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="text" >
         <string>results_score_cutoff</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QDoubleSpinBox" name="minScoreBox" >
        <property name="toolTip" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="statusTip" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="whatsThis" >
         <string>results_score_cutoff_tip</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="accelerated" >
         <bool>false</bool>
        </property>
        <property name="decimals" >
         <number>1</number>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>319</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="3" >
       <widget class="QSpinBox" name="evalueAsNSeqBox" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="statusTip" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="whatsThis" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum" >
         <number>2000000000</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="toolTip" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="statusTip" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="whatsThis" >
         <string>e_value_as_nsec_tip</string>
        </property>
        <property name="text" >
         <string>e_value_as_n_seq</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="domEvalueCuttofBox" >
        <property name="toolTip" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="statusTip" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="whatsThis" >
         <string>results_evalue_cutoff_tip</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="prefix" >
         <string>1E</string>
        </property>
        <property name="minimum" >
         <number>-99</number>
        </property>
        <property name="maximum" >
         <number>0</number>
        </property>
        <property name="value" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" >
       <spacer name="horizontalSpacer_5" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="3" >
       <widget class="QComboBox" name="algoCombo" >
        <property name="toolTip" >
         <string>algo_combo_tip</string>
        </property>
        <property name="statusTip" >
         <string>algo_combo_tip</string>
        </property>
        <property name="whatsThis" >
         <string>algo_combo_tip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="statusLabel" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>ok_button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>close_button</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>HMMSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>439</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel" >
     <x>244</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
